<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchResults</class>
 <widget class="QDialog" name="SearchResults">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Results</string>
  </property>
  <widget class="QComboBox" name="comboNames">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of shows:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Started:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEnd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ended:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>391</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGenre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchResults</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchResults</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
